integration/helium.md

```diff
@@ -1 +1,86 @@
+
+Once the integration is enabled any message sent by the end device will also be published to the TTN Mapper website. In order for TTN Mapper to correctly interpret the incoming messages, a payload decoder needs to be configured. 
+* If your device is sending data in Cayenne LPP format, choose this decoder. 
+* Otherwise create a custom javascript decoder which decoded the raw payload into one of these formats:
+  1. A JSON object containing the keys "**latitude**", "**longitude**" and "**altitude**". Some variations of these keys will also work. 
+  2. In addition the JSON object should preferably contain one of the following keys "**hdop**", "**accuracy**" or "**sats**".
+
+If you are developing your own GPS enabled LoRa device please check the following [Github repository](https://github.com/ttnmapper/gps-node-examples) for example end-device software and decoder functions to be used with them.
+
+## Setting up Helium for TTN Mapper
+
+Following steps shall be followed to setup Helium for TTN Mapper:
+
+Assuming you created a device and setup a payload decoder the meets the requirements as stated before:
+1. Create integration
+2. Create label and link devices
+3. Configure your TTN Mapper application in **Flows**
+
+### 1. Create the integration
+
+On Helium console, click on the *add (+)* icon in the bottom left and select **Add integrations**. Then click on HTTP. In the configuration page for the integration fill in the following:
+
+* **Endpoint URL (Required)**: Enter `https://integrations.ttnmapper.org/helium/v1`
+* **Step 3 - Name your Integration (Required)**: Enter the name for your integration
+* **Endpoint**: Add `https://integrations.ttnmapper.org/chirp/v3/events`
+* **HTTP Headers (Mandatory)**: Add header **TTNMAPPERORG-NETWORK** and set it to 'HELIUM'
+* **HTTP Headers (Optional)**: Add headers to the integration as specified below at **Headers**.
+* Click **Add Integration** to confirm and save your integration.
+
+#### Optional Headers
+
+* **TTNMAPPERORG-EXPERIMENT**: You only need to provide an experiment name if you are testing and do not want your mapping results to contribute to the main coverage map for the network. In other words, when you are mapping normally with a device that is between 1 and 2 metre above ground level, you can leave this field blank. If you are doing anything else like launching and tracking a balloon, please provide a unique experiment name here. Adding the date to the experiment name helps to find it again later. Also see [Experiments](#experiments).
+* **TTNMAPPERORG-USER**: Add your email address in this header to allow TTN Mapper to associate your coverage data with you. This will allow to easily find your data, and also verify when you request your data to be deleted.
```

Comment:
The user is a required header

Reply:
I will change bit chirpstack is not consistent with this statement ;-)

Reply:
Done